main_utils.py

```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+
+        This event works based on bot creation logic.
+        Please ensure you add your ticket bot or approved
```

Comment:
#30 

> 29 + "Bot creation logic" is a bit broad, maybe say "on ticket creation events"? (Assuming you are writing about what I think you are)
30 + "Ticket creation users", and "ticket users configuration" is left unexplained, do you mean the staff teams section of the main tickets' bot, or something else? If so, specify!